data/kimse2020Viral/kimse2020code.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import yaml
+import pandas as pd
+import numpy as np
+class folded_str(str): pass
+class literal_str(str): pass
+
+def folded_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='>')
+def literal_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
+
+yaml.add_representer(folded_str, folded_str_representer)
+yaml.add_representer(literal_str, literal_str_representer)
+
+
+# In[2]:
+
+
+data = pd.read_csv("CombinedDataset-2.csv") 
+data=data[113:132]
+
+
+# In[4]:
+
+
+data = data.sort_values(by=['Day','StudyNum'])
+
+
+# In[47]:
+
+
+data = data.replace({"Sex": {"M": "male", "F": "female"},
+                           "SevMax3": {"Moderate": "moderate", "Mild": "mild", "Severe": "severe"},
+                           "PatientID":{"4-1":"1", "4-2":"2","4-3":"3"}})
+print(data)
+
+
+# In[63]:
+
+
+df = pd.DataFrame(data) if not isinstance(data, pd.DataFrame) else data
+
+participants = []
+
+for patient_id, patient_data in df.groupby("PatientID"):
+#patient_id in df["PatientID"].unique():
+#     patient_data = df[df["PatientID"] == patient_id]
+    participant = {
+        "attributes": {
+            # "day": int(patient_data["Day"].iloc[0]),
+            "age": float(patient_data["Age"].iloc[0]),
+            "sex": str(patient_data["Sex"].iloc[0]),
+            # "estimated": int(patient_data["Estimated"].iloc[0]),
+            # "sevmax": float(patient_data["SevMax"].iloc[0]),
+        },
+        # "sev1st": float(patient_data["Sev1st"].iloc[0]),
+        "death": int(patient_data["Died"].iloc[0]),
```

Comment:
Let's move this to the `attributes` section and cast to a `bool` so it's either true or false.